documentation/assemblies/assembly-monitoring.adoc

```diff
@@ -4,20 +4,14 @@
 
 :parent-context: {context}
 [id='monitoring-{context}']
-= Monitoring {ProductName}
-
-Monitoring services run frequent health checks  on {ProductName} and send alerts when health checks
-fail. Health is assessed using Prometheus and `kube-state-metrics`. Alerting is implemented with
-`Alertmanager`. Grafana is also configured to provide a dashboard of the current status of health
-checks.
 
-include::../modules/proc-deploy-prometheus.adoc[leveloffset=+1]
+= Monitoring {ProductName}
 
-include::../modules/proc-deploy-kube-state-metrics.adoc[leveloffset=+1]
+Monitoring can be achieved by deploying inbuilt monitoring or using pre-existing monitoring infrastructure.
```

Comment:
Suggest: You can monitor {ProductName} by deploying built-in monitoring tools(?services?) or using your pre-existing monitoring infrastructure.
^Is my rewrite technically correct?